profiles/markaspot/themes/ntxuva/js/ntxuva.js

```diff
@@ -727,7 +727,8 @@ var lang_pt = {
             if (!this.inited) {
                 this.$yearSelect.append(yearsDD.join(''));
             }
-
+            //clear the list first
```

Comment:
I suggest you remove this comment. It is redundant, as bellow you're calling empty which is synonymous to clear.